packages/synapse-react-client/src/components/SetAccessRequirementCommonFields/SetAccessRequirementCommonFields.test.tsx

```diff
@@ -128,12 +139,68 @@ describe('SetAccessRequirementCommonFields', () => {
     await waitFor(() => {
       const managedAr: Pick<
         ManagedACTAccessRequirement,
-        'concreteType' | 'name' | 'subjectIds' | 'accessType'
+        | 'concreteType'
+        | 'name'
+        | 'subjectIds'
+        | 'accessType'
+        | 'subjectsDefinedByAnnotations'
       > = {
         concreteType: MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
         name: name,
         subjectIds: [entitySubject],
         accessType: ACCESS_TYPE.DOWNLOAD,
+        subjectsDefinedByAnnotations: false,
+      }
+
+      expect(createAccessRequirementSpy).toHaveBeenCalledWith(
+        MOCK_ACCESS_TOKEN,
+        managedAr,
+      )
+      expect(updateAccessRequirementSpy).not.toHaveBeenCalled()
+
+      expect(onSave).toHaveBeenCalledTimes(1)
+      expect(onSave).toHaveBeenLastCalledWith(
+        MOCK_NEWLY_CREATED_AR_ID.toString(),
+        MANAGED_ACT_ACCESS_REQUIREMENT_CONCRETE_TYPE_VALUE,
+      )
+      expect(onError).not.toHaveBeenCalled()
+    })
+  })
+
+  test('creates a new managed entity AR with subjects defined by annotation', async () => {
```

Comment:
Try flipping on the subjectsDefinedByAnnotationsCheckbox to verify that the field is set to true, and subjectIds array is now empty (they are mutually exclusive)

Reply:
What do you think about adding a test for editing an AR with `subjectsDefinedByAnnotations` to make sure that the content loads and `subjectsDefinedByAnnotationsCheckbox` is checked?

Reply:
Good idea.  Added this test